scripts/run.sh

```diff
@@ -65,13 +65,13 @@ fi
 # Output location
 BASEDIR=${DATADIR:-${PWD}}
 
-# XRD Write locations
-XRDWURL="xroots://dtn2201.jlab.org/"
+# XRD Write locations (allow for empty URL override)
+XRDWURL=${XRDWURL-"xroots://dtn2201.jlab.org/"}
 XRDWBASE=${XRDWBASE:-"/eic/eic2/EPIC"}
 
-# XRD Read locations
-XRDRURL="root://dtn-eic.jlab.org/"
-XRDRBASE="/work/eic2/EPIC"
+# XRD Read locations (allow for empty URL override)
+XRDRURL=${XRDRURL-"root://dtn-eic.jlab.org/"}
```

Comment:
Should there be a : in the middle
```XRDRURL=${XRDRURL:-"root://dtn-eic.jlab.org/"}```

Reply:
The `:-` means to replace both when empty and undefined. `-` means to replace only when undefined, not when defined as empty.